File-Formats/Presentation/FAQ.md

```diff
@@ -46,3 +46,56 @@ documentation: UG
    
    You can improve the quality of converted images by specifying the image resolution. Refer – [Converting PowerPoint presentation to Images](/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-images)
 
+10. ## What are the NuGet packages to be installed to perform PowerPoint Presentation to PDF conversion in Linux OS?
+
+In Linux OS, you can perform PowerPoint Presentation to PDF conversion using .NET Core (Targeting .netcoreapp) application. You can refer [PowerPoint Presentation to PDF conversion NuGet packages](https://help.syncfusion.com/file-formats/presentation/nuget-packages-required#converting-powerpoint-presentation-into-pdf) to know about the packages required to deploy .NET Core (Targeting .netcoreapp) applications with PowerPoint Presentation to PDF conversion capabilities.
```

Comment:
In Linux OS, perform PowerPoint Presentation to PDF conversion using the .NET Core (Targeting .netcoreapp) application. Refer to [PowerPoint Presentation to PDF conversion NuGet packages](https://help.syncfusion.com/file-formats/presentation/nuget-packages-required#converting-powerpoint-presentation-into-pdf) to know about the packages required to deploy the .NET Core (Targeting .netcoreapp) applications with PowerPoint Presentation to PDF conversion capabilities.